src/gsy_e/models/strategy/infinite_bus.py

```diff
@@ -44,15 +44,15 @@ def __init__(self, energy_sell_rate=None, energy_rate_profile=None, energy_buy_r
         # buy
         if all(arg is None for arg in [
                buying_rate_profile, buying_rate_profile_uuid, energy_buy_rate]):
-            energy_buy_rate = ConstSettings.GeneralSettings.DEFAULT_MARKET_MAKER_RATE
+            energy_buy_rate = GlobalConfig.FEED_IN_TARIFF
         self._buy_energy_profile = EnergyProfile(
             buying_rate_profile, buying_rate_profile_uuid, energy_buy_rate,
             profile_type=InputProfileTypes.IDENTITY)
 
         # sell
         if all(arg is None for arg in [
                energy_rate_profile, energy_rate_profile_uuid, energy_sell_rate]):
-            energy_sell_rate = ConstSettings.GeneralSettings.DEFAULT_MARKET_MAKER_RATE
+            energy_sell_rate = GlobalConfig.MARKET_MAKER_RATE
```

Comment:
Why not keep using the default here? The MARKET_MAKER_RATE is mutable and changes throughout the simulation.